site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -160,34 +174,143 @@ Response:
 {"result":{"object_meta":{"name":"local","namespace":"default","uid":"1234","resource_version":"v1","generation":"1","creation_timestamp":"1533766607","annotations":{"annotation":"true"},"labels":{"islocal":"true"}},"status":{"state":"Ready","address":"127.0.0.1","ports":[{"name":"default","port":7777}]}}}
 {"result":{"object_meta":{"name":"local","namespace":"default","uid":"1234","resource_version":"v1","generation":"1","creation_timestamp":"1533766607","annotations":{"annotation":"true"},"labels":{"islocal":"true"}},"status":{"state":"Ready","address":"127.0.0.1","ports":[{"name":"default","port":7777}]}}}
 ```
+### Metadata Management
 
-### Reserve
+#### Set Label
 
-Move Gameserver into a Reserved state for a certain amount of seconds for the future allocation.
+Apply a Label with the prefix "agones.dev/sdk-" to the backing `GameServer` metadata. 
 
-- Path: `/reserve`
-- Method: `POST`
-- Body: `{"seconds": "5"}`
+See the SDK [SetLabel]({{< ref "/docs/Guides/Client SDKs/_index.md#setlabel-key-value" >}}) documentation for restrictions.
 
-#### Example
+##### Example
 
 ```bash
-$ curl -d '{"seconds": "5"}' -H "Content-Type: application/json" -X POST http://localhost:${AGONES_SDK_HTTP_PORT}/reserve
+$ curl -d '{"key": "foo", "value": "bar"}' -H "Content-Type: application/json" -X PUT http://localhost:${AGONES_SDK_HTTP_PORT}/metadata/label
 ```
 
-### Allocate
+#### Set Annotation
 
-With some matchmakers and game matching strategies, it can be important for game servers to mark themselves as `Allocated`.
-For those scenarios, this SDK functionality exists. 
+Apply a Annotation with the prefix "agones.dev/sdk-" to the backing `GameServer` metadata
```

Comment:
```suggestion
Apply an Annotation with the prefix "agones.dev/sdk-" to the backing `GameServer` metadata
```
or the Annotation not sure what suits better here.

Reply:
Done. PTAL!